WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -121,185 +139,95 @@ - (void)viewDidLoad
-            if (self.blog.url != nil) {
-                self.urlTextField.text = self.blog.url;
-                
-                // Make a margin exception for URLs since they're so long
-                cell.minimumLabelWidth = EditSiteURLMinimumLabelWidth;
-            } else {
-                self.urlTextField.text = @"";
+        case SiteSettingsAccountUsername: {
+            if (self.usernameTextCell) {
+                return self.usernameTextCell;
             }
-            
-            self.urlTextField.enabled = [self canEditUsernameAndURL];
-            [WPStyleGuide configureTableViewTextCell:cell];
-            
-            return cell;
-        } break;
-        case EditSiteRowUsername: {
-            WPTextFieldTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:TextFieldCellIdentifier];
-            
-            cell.textLabel.text = NSLocalizedString(@"Username", @"Label for entering username in the username field");
-            self.usernameTextField = cell.textField;
-            self.usernameTextField.placeholder = NSLocalizedString(@"Enter username", @"(placeholder) Help enter WordPress username");
-            [self.usernameTextField addTarget:self action:@selector(showSaveButton) forControlEvents:UIControlEventEditingChanged];
-            [self configureTextField:self.usernameTextField asPassword:NO];
-            if (self.blog.usernameForSite != nil) {
-                self.usernameTextField.text = self.blog.usernameForSite;
+            self.usernameTextCell = [[SettingTableViewCell alloc] initWithStyle:UITableViewCellStyleValue1 reuseIdentifier:SettingCellIdentifier];
+            self.usernameTextCell.textLabel.text = NSLocalizedString(@"Username", @"Label for entering username in the username field");
+            if (self.blog.usernameForSite) {
+                self.usernameTextCell.detailTextLabel.text = self.blog.usernameForSite;
             } else {
-                self.usernameTextField.text = @"";
+                self.usernameTextCell.detailTextLabel.text = NSLocalizedString(@"Enter username", @"(placeholder) Help enter WordPress username");
             }
-            
-            self.usernameTextField.enabled = [self canEditUsernameAndURL];
-            [WPStyleGuide configureTableViewTextCell:cell];
-            
-            return cell;
+            [WPStyleGuide configureTableViewCell:self.usernameTextCell];
+            self.usernameTextCell.accessoryType = UITableViewCellAccessoryNone;
+            self.usernameTextCell.selectionStyle = UITableViewCellSelectionStyleNone;
+            return self.usernameTextCell;
         } break;
-        case EditSiteRowPassword: {
-            WPTextFieldTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:TextFieldCellIdentifier];
-            
+        case SiteSettingsAccountPassword: {
+            WPTextFieldTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:PasswordCellIdentifier];
```

Comment:
Since the text field isn't going to be editable, maybe we should use `SettingTableViewCell` here as well